egs2/chime7/task1/sub_track1.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+# main stages
+stage=1
+stop_stage=1
+
+
+# NOTE, use absolute paths !
+chime7_root=${PWD}/chime7_task1
+chime5_root= 
+chime6_root=/raid/users/popcornell/CHiME6/espnet/egs2/chime6/asr1/CHiME6 # will be created automatically from chime5
+# but if you have it already you can use your existing one.
+dipco_root=/raid/users/popcornell/CHiME6/DipCO/DiPCo/ # this will be automatically downloaded
+mixer6_root=/raid/users/popcornell/mixer6/
+manifests_root=./data/lhotse
+
+# dataprep options
+cmd_dprep=run.pl
+dprep_stage=2
+gss_dump_root=./exp/gss
+ngpu=4  # set equal to the number of GPUs you have, used for GSS and ASR training
+
+tr_dsets=train
+cv_dsets=dev
+tt_dsets=
+
+. ./path.sh
+. ./cmd.sh
+. ./utils/parse_options.sh
+
+# gss config
```

Comment:
I reformatted most of the code.  
I don't agree however to move everything to `local/data.sh` and keep ASR only in the `run.sh` in the ESPNet-style.
 
Some reasons:
1. **Legacy with CHiME-6 recipe** : Kaldi does it that way, having all stages in the main in plain view. I don't see any advantage of moving everything in what then becomes an "hidden" main (data.sh will do most of the work). 
Any reason behind this than simply adhering to ESPNet-convention ? 
2. **More simple to me:** users can see all the steps in one script and the options are all in run.sh both for ASR and GSS, otherwise I have to pass many options to the data.sh (GSS + all the data prep steps). 

Again I think I can move ASR data preparation in `local/data.sh`.

Reply:
1.
As I said, it becomes the local processing **independent**

2.
`run.sh` does not become simple

Reply:
probably, you misunderstood some parts.
I'll explain my idea later.

Reply:
Yeah thanks for the clarification I totally misunderstood it